src/modsadmins/mods.ts

```diff
@@ -7,6 +7,7 @@ export const modsArray = [
   'goofy', // jams
   'sadnixon',
   'fossa',
+  '1',
```

Comment:
Let's not commit this lol. Try https://stackoverflow.com/questions/7335420/global-git-ignore